narwhals/_pandas_like/utils.py

```diff
@@ -10,12 +10,6 @@
 from typing import Sequence
 from typing import TypeVar
 
-from narwhals._arrow.utils import (
-    narwhals_to_native_dtype as arrow_narwhals_to_native_dtype,
-)
-from narwhals._arrow.utils import (
-    native_to_narwhals_dtype as arrow_native_to_narwhals_dtype,
-)
```

Comment:
these imports were the issue preventing us from doing this before

by moving them to only where they're needed, we can have more safe top-level imports